site/releases/2024-dec-24/release-notes.qmd

```diff
@@ -0,0 +1,413 @@
+
+::: {.w-70-ns}
+You can now gate edit access to specific model inventory fields to certain roles via the **[write permissions]{.smallcaps}** setting.
+
+:::
+
+::: {.w-30-ns .tc}
+[Manage model inventory fields](/guide/model-inventory/manage-model-inventory-fields.qmd){.button}
+
+:::
+
+::::
+
+- Leave this field blank to leave permissions as default.
+- By default, model owners and validators have write access. Model developers have read-only access.[^4]
+
+## Bug fixes
+
+### {{< var validmind.platform >}} (v1.27.17)
+<!---
+PR #1106: Deleting a finding should remove it from a validation report
+URL: https://github.com/validmind/frontend/pull/1106
+Labels: bug
+--->
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+- Previously, deleting a finding did not remove it from linked validation reports. 
+- Now, deleting a finding will show a confirmation warning and subsequently remove it from any linked reports.
+
+::: {.tc}
+[Add and manage model findings](/guide/model-validation/add-manage-model-findings.qmd#delete-model-findings){.button}
+
+:::
+
+:::
+
+::: {.w-40-ns .tc}
+![Deletion confirmation dialogue for a finding linked to a validation report](delete-linked-finding.png){fig-alt="A screenshot of the deletion confirmation dialogue for a finding linked to a validation report" .screenshot}
+
+:::
+
+::::
+
+## Documentation 
+
+### Code samples
+
+#### Initialize {{< var vm.product >}} with stored credentials
+
+We've included details on how to initialize {{< var vm.product >}} with credentials stored in an `.env` file in our Jupyter Notebook samples.
+
+:::: {.flex .flex-wrap .justify-around}
+
+::: {.w-60-ns}
+Our documentation guides have also been updated to match this new experience:
+
+:::
+
+::: {.w-40-ns .tc}
+[Store model credentials in `.env` files](/developer/model-documentation/store-credentials-in-env-file.qmd){.button}
+
+:::
+
+::::
+
+<!---
+PR #543: Add developer fundamentals videos & training fixes
+URL: https://github.com/validmind/documentation/pull/543
+Labels: documentation
+--->
+### User guide updates
+
+#### Developer Fundamentals video series
+
+We created a series of 10 short videos to help you learn about the model documentation process as a developer:
+
+- These videos introduce you to generating model documentation that includes tests. 
+- You will also learn about adding your own tests, editing the content online, and then submitting your documentation for approval so that a validator can review your submitted documentation.
+
+{{< video https://www.youtube.com/embed/videoseries?si=aP7yLOGO60h4vi8C&amp;list=PL7I7ZrCoVCCP876aLRCjitZqYif96Whlz title='Developer Fundamentals' >}}
+
+<!---
+PR #552: Updated workflows/actions to dynamically generate an `.env` file
+URL: https://github.com/validmind/documentation/pull/552
+Labels: documentation
+--->
+
+<!-- https://github.com/validmind/documentation/pull/567 -->
+
+### {{< var validmind.training >}} updates
+
+##### Example executed Jupyter Notebook
```

Comment:
dont know if this was on purpose, but the link to the Developer Fundamentals shows up on "side" of the screen in the preview - display glitch? 

![Screenshot 2024-12-20 at 12 26 12 PM](https://github.com/user-attachments/assets/a7b3de4f-8977-4aa0-9b9e-c0df6ec651c6)

Reply:
Yes, this is on purpose — it looked better in the margin than as another"image" in the body,